renderer/components/Channels/ChannelCreateForm.js

```diff
@@ -195,14 +195,14 @@ class ChannelCreateForm extends React.Component {
     const [pubkey, host] = nodePubkey.split('@')
 
     // Determine the fee rate to use.
-    const satPerByte = this.getFee()
+    const satPerVByte = this.getFee()
```

Comment:
I don't think you can do this, because `getFee` is not returning `satPerVByte` - its returning `satPerByte` which is a different unit to `satPerVByte`

Reply:
Done, reverted.